### PR TITLE
Contactmeform

### PR DESCRIPTION
**Pull request by ecekpo**
Added modified contactmeform work.

### CODE REVIEW

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @ecekpo @Dev-inspired @Ngufor-emmanuella 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Please fix the eslint errors.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -160,14 +158,104 @@ <h1 class="multipost_text">Multi-post <br> stories</h1>
...
+                <footer>
+                    <div class="Background-image">
+                        <h4 class="Contact-me">Contact me</h4>
+                        <p class="contact-details">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it </p>
+                           <form action="https://formspree.io/f/moqrrblw" method="post">
+                 
+                          <input type="text" name="username" id="name" placeholder="Your name" maxlength="30" required/>
+                          <input type="email" name="email" id="email" placeholder="Your email id" required/>
+                          <textarea name="message" id="message" placeholder="Type your message" rows="8" maxlength="500"></textarea>
+                          <button type="submit" id="contact-btn">Get in touch</button>
```
Comment (tufoinnkuo10) [inline index.html:251]: - [ ] kindly add the hover interaction on the button and add the size so it looks like the figma version.

File: index.html [hunk 2]
```diff
@@ -160,14 +158,104 @@ <h1 class="multipost_text">Multi-post <br> stories</h1>
...
                           </div>
+                
                       </div>
-                    </section>                              
+                  </div>
+                </section>
+
+                <footer>
+                    <div class="Background-image">
+                        <h4 class="Contact-me">Contact me</h4>
```
Comment (tufoinnkuo10) [inline index.html:244]: - [ ] please use the correct background image for the contact section

File: README.md [hunk 3]
```diff
@@ -44,6 +44,18 @@ To get a local copy up and running follow these simple example steps.
...
+
+- GitHub: [@githubhandle](https://github.com/ecekpo)
+- Twitter: [@twitterhandle](https://twitter.com/@ememcookey)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/emem-ekpo-857135234/)
+
+**Author3**
+
+- GitHub: [@githubhandle](https://github.com/ecekpo)
+- Twitter: [@twitterhandle](https://twitter.com/@ememcookey)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/emem-ekpo-857135234/)
```
Comment (Kakalanp) [inline README.md:57]: [OPTIONAL]

Consider removing unnecessary fields on your README file, remember that this file is often your first chance to make you project look professional :briefcase:

**Review by Kakalanp — APPROVED** [review]
# STATUS: APPROVED :green_circle: 

### Your project is approved @ecekpo! :tada: 

There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :clinking_glasses: 

almost there

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**